File: Logitude/JustWebFreight/WebFreight.Web/DataContracts/LTBResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebFreight.Web.DataContracts
{
    public class LTBResponse
    {
        public decimal? EndBalanceForeign { get; set; }
        public decimal EndBalanceLocal { get; set; }
        public string Have1CurrencyIdInPeriod { get; set; }
        public DateTime? MaxCreateAt { get; set; }
        public decimal? StartBalanceForeign { get; set; }
        public decimal StartBalanceLocal { get; set; }
        public int? TotalRowCount { get; set; }
        public bool SuppressCumulativeDueMultiCurrencyInPeriod { get; set; }
                
    }
}

File: Logitude/AngularModules/AngularModules/Accounting/DataContracts/LTBResponse.ts
export class LTBResponse {
    EndBalanceForeign: number;
    EndBalanceLocal: number;
    Have1CurrencyIdInPeriod: string;
    MaxCreateAt: Date;
    StartBalanceForeign: number;
    StartBalanceLocal: number;
    TotalRowCount: number;
    SuppressCumulativeDueMultiCurrencyInPeriod: boolean;
    StartBalanceForeignList: BalanceCurrency[];
    EndBalanceForeignList: BalanceCurrency[];
}

export class BalanceCurrency {
    CurrencyId: string;
    CurrencyCode: string;
    CurrencySign: string;
    BalanceForeign: string;
    BalanceLocal: string;
}
